libmscore/segmentlist.h

```diff
@@ -1,82 +1,81 @@
-//=============================================================================
-//  MuseScore
-//  Music Composition & Notation
+// =============================================================================
+// MuseScore
+// Music Composition & Notation
 //
-//  Copyright (C) 2011 Werner Schweer
+// Copyright (C) 2011 Werner Schweer
 //
-//  This program is free software; you can redistribute it and/or modify
-//  it under the terms of the GNU General Public License version 2
-//  as published by the Free Software Foundation and appearing in
-//  the file LICENCE.GPL
-//=============================================================================
+// This program is free software; you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 2
+// as published by the Free Software Foundation and appearing in
+// the file LICENCE.GPL
+// =============================================================================
 
 #ifndef __SEGMENTLIST_H__
 #define __SEGMENTLIST_H__
```

Comment:
not `MU_{MODULE}_{CLASSNAME}_H` as per the new coding style?

Reply:
I think this needs to be fixed later, in a separate pull request

Reply:
This PR only addresses the indents issue. Other changes are recommended to follow when one creates new PR.
Anyway, I would be happy to listen to arguments why that or those suggestion from the new code style is wrong. It is better to change the code style earlier, you know :)

Reply:
Not sure we will have files in different modules with the same names. Like inspector/note.h and libmscore/note.h? I'm not sure. I would leave it as it is with just a capitalized filename.